src/tabs.test.jsx

```diff
@@ -61,4 +58,15 @@ describe('Tabs', function() {
 		expect(tabsUlEl.classList).not.toContain('tabs--bordered');
 	});
 
+	it('appropriately adds passed prop classname to component `Tabs` component', function() {
+		const tab = TestUtils.findRenderedDOMComponentWithClass(componentBasic, 'test-class-wrapper-name');
+		expect(() => tab).not.toThrow();
+		expect(tab.tagName).toBe('NAV');
+	});
+
+	it('appropriately adds passed prop classname to component `TabsTab` component', function() {
+		const tabsTab = TestUtils.findRenderedDOMComponentWithClass(componentBasic, 'test-class-name');
```

Comment:
set these classes as constants to make it easier to link the test input with the test output